src/commands/deploy/internal/executeDeployScriptSas9.ts

```diff
@@ -36,10 +36,7 @@ export async function executeDeployScriptSas9(
       authConfigSas9: { userName, password }
     })
     .catch((err) => {
-      process.logger?.log(formatErrorString(err))
```

Comment:
Error thrown from sasjs adapter already contains the proper message for missing of sasjs runner. Therefore, its enough to print formatted error string. Also this statement would have never been executed because errorCode thrown from adapter  is 500 in case of missing of sasjs runner.